command/core.py

```diff
@@ -46,3 +50,23 @@ def handle_team_join(self, event_data):
             logging.info(new_id + " added to database - user notified")
         except SlackAPIError:
             logging.error(new_id + " added to database - user not notified")
+
+    def get_help(self):
+        """Get help messages and return a formatted string for messaging."""
+        message = {"text": "Displaying all available commands. To read about"
+                           " a specific command, use `/rocket COMMAND help`\n",
+                   "mrkdwn": "true"}
+        attachments = []
+        for cmd in self.__commands:
+            cmd_name = self.__commands[cmd].get_name()
+            cmd_text = "*[" + cmd_name.upper() + "]*\n\n" + \
+                "Commands for editing", cmd_name + "s."
```

Comment:
Out of curiosity, what's the `"s."` for?

Reply:
It is to make the names plural, e.g. It would look like "commands for editing users, commands for editing teams

Reply:
Ooh I see, that's clever. One thing to keep in mind there, though, is that not all commands will necessarily follow that naming convention in the future (though it's probably fine for now).